thorlcr/graph/thgraphmaster.cpp

```diff
@@ -1485,7 +1485,9 @@ CJobMaster::CJobMaster(IConstWorkUnit &_workunit, const char *graphName, ILoaded
     init();
 
     Owned<IProperties> traceHeaders = extractTraceDebugOptions(workunit);
-    Owned<ISpan> requestSpan = queryTraceManager().createServerSpan(workunit->queryWuid(), traceHeaders);
+    Owned<ISpan> requestSpan = queryTraceManager().createServerSpan("run_graph", traceHeaders);
+    requestSpan->setSpanAttribute("hpcc.wuid", workunit->queryWuid());
```

Comment:
It feels like some of the most common attributes should be defined in JTrace

Reply:
This is something I'm still not sure about.  I can see the argument for having common strings - it catches typing mistakes, but I also find the code harder to read.

I suspect I could be persuaded for strings like this that are likely to be scattered through the code.